pyproject.toml

```diff
@@ -31,12 +31,12 @@ classifiers = [
 protool = 'protool:command_line.run'
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-pyOpenSSL ="^19.0.0"
+python = "^3.7"
```

Comment:
I ran into this by chance yesterday. You may need to set this to `^3.7.2` to ensure Pylint works when it gets updated. (Might be worth doing all dev dependencies in on go)

Reply:
Update all the dev-dependencies and fix lint issues, please take another look, thanks.